### ISSUE TITLE
Clean or replace hand-drawn pictures

### ISSUE THREAD

**Issue by mschauer** [mschauer/CausalInference.jl#63]
in https://mschauer.github.io/CausalInference.jl/latest/examples/backdoor_example/ (not against hand-drawn, actually, but they are a bit difficult to read)

**Comment by mschauer** [comment]
However you like, I would use Inkscape with latex plugin or TikZ to do it by hand.

**Comment by alonsoC1s** [comment]
Sorry for taking such a long time. I wanted to get more confortable with TikZ.

Here is an example of a replacement diagram, I would like to get some feedback on it.

![Screen Shot 2022-01-06 at 9 10 41](https://user-images.githubusercontent.com/12630090/148404520-bda1fe6f-6353-46e7-9a8a-3b2ef63535a4.png)

**Comment by mschauer** [comment]
A nice, thank you. We should perhaps replace the numbers by letters. Make it "A, B, ..., H" (capital letters) instead of "1, 2, 3, 4, 5, 6, 7, 8"?

**Comment by alonsoC1s** [comment]
Awesome. I'll clean up the one above and start working on the other diagrams

**Comment by alonsoC1s** [comment]
I made the adjustments and worked on the missing diagrams. Here's a look. Let me know if there is anything left to do to make them clearer or if I made any mistakes.
![Screen Shot 2022-01-11 at 19 58 45](https://user-images.githubusercontent.com/12630090/149050605-f22e95db-790c-43d3-aa98-dc29de04edce.png)
![Screen Shot 2022-01-11 at 19 58 55](https://user-images.githubusercontent.com/12630090/149050608-3e6f15a3-7365-4a6c-ab8b-6edd6586d921.png)
![Screen Shot 2022-01-11 at 19 59 03](https://user-images.githubusercontent.com/12630090/149050609-a7e3750d-3ba0-4e58-bfb5-53521352bfb1.png)

They are ordered exactly as they can be found on the documentation.
